tests/riscv64/smoke.sh

```diff
@@ -35,40 +37,39 @@ function generate_llvm_cpu_vmfb {
     )
   elif [[ "${target}" == "tosa" ]]; then
     local input_file="${1}"; shift
-    iree-import-tflite -o "${BUILD_RISCV_DIR?}/tosa.mlir" "${input_file}"
+    "${IREE_IMPORT_TFLITE_BIN}" -o "${BUILD_RISCV_DIR}/tosa.mlir" "${input_file}"
     compile_args+=(
       --iree-input-type=tosa
       --iree-llvm-target-cpu-features="+m,+a,+f,+d,+c"
-      "${BUILD_RISCV_DIR?}/tosa.mlir"
+      "${BUILD_RISCV_DIR}/tosa.mlir"
     )
   elif [[ "${target}" == "tosa-rvv" ]]; then
     local input_file="${1}"; shift
-    iree-import-tflite -o "${BUILD_RISCV_DIR?}/tosa.mlir" "${input_file}"
+    "${IREE_IMPORT_TFLITE_BIN}" -o "${BUILD_RISCV_DIR}/tosa.mlir" "${input_file}"
     compile_args+=(
       --iree-input-type=tosa
       --iree-llvm-target-cpu-features="+m,+a,+f,+d,+c,+v"
       --riscv-v-fixed-length-vector-lmul-max=8
       --riscv-v-vector-bits-min=256
```

Comment:
```suggestion
      --riscv-v-vector-bits-min=512
```

Reply:
I'd prefer to leave these sorts of substantive changes out of PRs related to moving build systems. Then someone more familiar with riscv can change things as approparite

Reply:
Sure. I will send out the clean up PR after the migration.